Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -616,8 +616,11 @@ GstBuffer* gstBufferNewWrappedFast(void* data, size_t length)
 GstElement* makeGStreamerElement(const char* factoryName, const char* name)
 {
     auto* element = gst_element_factory_make(factoryName, name);
-    if (!element)
+    if (!element) {
         WTFLogAlways("GStreamer element %s not found. Please install it", factoryName);
+        GST_WARNING("GStreamer element %s not found. Please install it", factoryName);
+        ASSERT_NOT_REACHED();
```

Comment:
Also I thought there was agreement about this function not asserting, because it's call sites are supposed to handle NULL return values. This is for optional elements, basically.